### PR TITLE
Add full movie view when requesting imdb from movies page

### PR DESCRIPTION
**Pull request by DariusIII**

### CODE REVIEW

**Comment by ruhllatio** [comment]
I'm guessing there is more to be committed?  A cursory check shows that viewmoviefull.tpl doesn't exist in Alpha.  So this would likely cause breakage.

**Comment by scottia** [comment]
Nice work on Gamma.. Little thing.  Clicking cover OR cover button has mixed results on what's displayed.  Also no Movie description on some.

File: www/themes/Gamma/templates/frontend/profileedit.tpl [hunk 1]
```diff
@@ -55,9 +55,7 @@
 		<div class="control-group">
 			<label class="control-label">Change site theme</label>
 			<div class="controls">
-				{if $page->settings->getSetting('userselstyle') == 1}
-					{html_options id="style" name='style' values=$themelist output=$themelist selected=$user.style}
-				{/if}
+				{html_options id="style" name='style' values=$themelist output=$themelist selected=$user.style}
```
Comment (niel) [inline www/themes/Gamma/templates/frontend/profileedit.tpl]: The profile.tpl needs to display this too.

Comment (DariusIII) [inline www/themes/Gamma/templates/frontend/profileedit.tpl]: Pushed the suggested changes.

File: www/themes/Gamma/templates/frontend/profile.tpl [hunk 2]
```diff
@@ -54,7 +58,7 @@
 		<th>Grabs Total:</th>
 		<td>{$user.grabs}</td>
 	</tr>
-	{if ($user.id==$userdata.id || $userdata.role==2) && $site->registerstatus==1}
+	{if (!$publicview || $isadmin) && $site->registerstatus==1}
```
Comment (niel) [inline www/themes/Gamma/templates/frontend/profile.tpl]: I'm not sure about the !$publicview. I'm reading this as if NOT public or is admin user, which doesn't make sense to me. Shouldn't it be if IS public or IS admin user?

File: www/themes/charisma/templates/frontend/profile.tpl [hunk 3]
```diff
@@ -105,7 +127,7 @@
 															{/if}
 															</tbody>
 														</table>
-														{if ($user.id==$userdata.id || $isadmin) && $site->registerstatus==1}
+														{if (!$publicview || $isadmin) && $site->registerstatus==1}
```
Comment (niel) [inline www/themes/charisma/templates/frontend/profile.tpl]: Same as for Gamma version of this file.

File: www/themes/omicron/templates/frontend/profile.tpl [hunk 4]
```diff
@@ -105,7 +127,7 @@
 															{/if}
 															</tbody>
 														</table>
-														{if ($user.id==$userdata.id || $isadmin) && $site->registerstatus==1}
+														{if (!$publicview || $isadmin) && $site->registerstatus==1}
```
Comment (niel) [inline www/themes/omicron/templates/frontend/profile.tpl]: Same as for Gamma version of this file.

File: www/themes/Gamma/templates/frontend/profile.tpl (hunk 2 above)
Comment (DariusIII) [inline www/themes/Gamma/templates/frontend/profile.tpl]: https://github.com/nZEDb/nZEDb/blob/master/www/themes/alpha/templates/frontend/profile.tpl#L44

It is same as in alpha.

File: www/themes/charisma/templates/frontend/profile.tpl (hunk 3 above)
Comment (DariusIII) [inline www/themes/charisma/templates/frontend/profile.tpl]: It is already fixed in newznab-tmux, just did not update files for this PR.

File: www/themes/omicron/templates/frontend/profile.tpl (hunk 4 above)
Comment (DariusIII) [inline www/themes/omicron/templates/frontend/profile.tpl]: It is already fixed in newznab-tmux, just did not update files for this PR.

File: www/themes/charisma/templates/frontend/profile.tpl (hunk 3 above)
Comment (DariusIII) [inline www/themes/charisma/templates/frontend/profile.tpl]: Same comment as for Gamma :)

File: www/themes/omicron/templates/frontend/profile.tpl (hunk 4 above)
Comment (DariusIII) [inline www/themes/omicron/templates/frontend/profile.tpl]: Same comment as for Gamma :)

File: www/themes/Gamma/templates/frontend/profile.tpl (hunk 2 above)
Comment (niel) [inline www/themes/Gamma/templates/frontend/profile.tpl]: ok. I'll merge it as is, but I'm still unsure about it. May be alpha is wrong too ;-)

**Pull request merged**